src/main/java/org/tron/peer/PeerBuilder.java

```diff
@@ -19,22 +23,25 @@
     private Wallet wallet;
     private ECKey key;
     private String type;
+    private Injector injector;
 
-    public PeerBuilder() {
-        Server.serverRun();
```

Comment:
I removed this because starting a server should not be done within a factory class. Starting the server is now being done in `org.tron.example.Tron.run()`